app/lib/utils/btc.js

```diff
@@ -84,7 +84,7 @@ export function satoshisToBits(satoshis) {
 export function satoshisToMillisatoshis(satoshis) {
   if (isEmptyAmount(satoshis)) return null
 
-  return satoshisToBits(satoshis) * 1000 || 0
+  return satoshis * 1000 || 0
```

Comment:
I wonder, why do we return `null` if amt is empty but use fallback ` || 0 `when something happens to `satoshis * 1000` ?
Probably not a big deal, but for consistency sake.

Reply:
TBH, the fallback value probably isn't required here at all. I added it for the sake of consistency with the rest of this module since all the other methods do it that way and I don't want to extend the scope of this package to try and change that since personally I think this entire file needs to be reworked at some point.